lib/DDG/Util/SpiceConstants.pm

```diff
@@ -0,0 +1,9 @@
+package DDG::Util::SpiceConstants;
+
+use strict;
+use warnings;
+
+# regex for allowed TLDS (grabbed from DDG core repo, in /lib/DDG/Util/Constants.pm)
```

Comment:
here it was! https://github.com/duckduckgo/zeroclickinfo-spice/commit/a75999c383e7a1a068f409af2d779221f762148a#diff-370547fe6407e842a2a7cc070b084393L25